### PR TITLE
Improve Wallet loading time, fix checking if lockup exists, reset Redux state on switch, recover, create account

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

File: src/components/profile/Profile.js [hunk 1]
```diff
@@ -141,6 +141,7 @@ export function Profile({ match }) {
                 await dispatch(loadRecoveryMethods())
                 dispatch(getAccessKeys(accountId))
                 dispatch(getLedgerKey())
+                await dispatch(getBalance())
```
Comment (vgrichina) [inline src/components/profile/Profile.js:144]: @marcinbodnar [not a blocker for this PR]: seems like we can improve even further if parallelise kicking off all the queries vs using `await` to do some of them sequentially? Basically kick off loading balance when we know that we need to do it vs when recovery methods already loaded, etc

File: src/utils/wallet.js [hunk 2]
```diff
@@ -183,7 +184,7 @@ class Wallet {
                         break
                     }   
                 }
-                this.selectAccount(nextAccountId)
+                store.dispatch(selectAccount(nextAccountId))
```
Comment (vgrichina) [inline src/utils/wallet.js:187]: @marcinbodnar do I understand correctly that larger plan is to break down `wallet.js` into Redux actions?

File: src/components/profile/Profile.js (hunk 1 above)
Comment (marcinbodnar) [reply src/components/profile/Profile.js:144]: @vgrichina Right, in this particular case I wanted to load Recovery Methods before fetching for staking balance details, which might take few seconds.

File: src/utils/wallet.js (hunk 2 above)
Comment (marcinbodnar) [reply src/utils/wallet.js:187]: @vgrichina Yes, that's the larger plan - in this case, we want to clear the account redux state while changing account.

**Pull request merged**